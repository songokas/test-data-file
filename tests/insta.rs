use insta::assert_yaml_snapshot;
use test_data_file::test_data_file;

fn split_words(s: &str) -> Vec<&str> {
    s.split_whitespace().collect()
}

#[test_data_file(kind = "csv", path = "tests/samples/snapshot")]
#[test]
fn test_split_words(name: String, words: String) {
    assert_yaml_snapshot!(name, split_words(&words));
}
