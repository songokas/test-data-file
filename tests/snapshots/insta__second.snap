---
source: tests/insta.rs
expression: split_words(&words)
snapshot_kind: text
---
- dear
- marry
